docs/sources/shared/upgrade/upgrade-common-tasks.md

```diff
@@ -115,7 +115,7 @@ To upgrade Grafana running on CentOS or RHEL, complete the following steps:
 
 1. Perform one of the following steps based on your installation.
 
-   - If you [downloaded an RPM package](https://grafana.com/grafana/download) to install Grafana, then complete the steps documented in [Install on RPM-based Linux]({{< relref "../../setup-grafana/installation/rpm/" >}}) to upgrade Grafana.
+   - If you [downloaded an RPM package](https://grafana.com/grafana/download) to install Grafana, then complete the steps documented in [Install Grafana on Red Hat, RHEL, or Fedora]({{< relref "../../setup-grafana/installation/redhat-rhel-fedora/" >}}) or [Install Grafana on SUSE or OpenSUSE]({{< relref "../../setup-grafana/installation/suse-opensuse/" >}}) to upgrade Grafana.
```

Comment:
Update the section header and intro sentence, we may also want to provide the `sudo zypper update` command for OpenSUSE/SUSE users since `sudo yum update` won't work for them.

Reply:
updated.